docs/pages/api-reference/operators.md

```diff
@@ -104,7 +125,40 @@ The `rename` operator has the following parameters:
 1. `columns: Dict[str, str]` - positional argument, that specifies the mapping of old column names to new column names.
 
 :::info
-Fennel does not permit column name collisions post the rename operation. An error is thrown if the new column names are already present in the dataset.
+Fennel does not permit column name collisions post the rename operation. An error is thrown if the new column names are
+already present in the dataset.
+:::
+
+### Dedup
+
+Fennel allows you to discard duplicate entries from a dataset by using the `dedup` operator. A subset of columns can be
+used for identifying duplicates by specifying the `by` parameter.
+
+The `dedup` operator has the following parameters:
+
+1. `by: List[str]` - optional kwarg that specifies the list of fields to use for identifying duplicates. If not
+   specified, all the columns (except the timestamp column) are used for identifying duplicates.
+
+::info
+The `dedup` operator can only be applied on datasets without keys
+:::
+
+### Explode
+
+Fennel allows you to explode `List` type columns into separate rows by using the `explode` operator.
+
+The `explode` operator has the following parameters:
+
+1. `columns: List[str]` - list of columns to explode. All the columns should be of type `List`. If the input column's
+   type if `List[T]`, the output column is assigned the type `Optional[T]`.
```

Comment:
Why does it get Optional[T] and not T?

Also, what happens when different input lists are of unequal lengths?

Reply:
If the lists are empty, the returned row will have nulls. This is similar to behavior in Pandas.

Reply:
It's an error if the different lists are of unequal length. This is also similar to Pandas: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.drop_duplicates.html